packages/ubuntu_init/lib/src/init_wizard.dart

```diff
@@ -54,6 +56,18 @@ class InitWizard extends ConsumerWidget {
             return Routes.initial;
           },
         ),
+        Routes.error: WizardRoute(
+          builder: (_) => const ErrorPage(),
+          userData: const WizardRouteData(
+            hasPrevious: false,
+            hasNext: false,
+          ),
+          onNext: (_) async {
+            final window = YaruWindow.of(context);
+            await window.close();
+            return Routes.initial;
```

Comment:
Shouldn't this return null?

Reply:
Doesn't really make a difference since we're closing the window - I just copied that part from the final route in the wizard. We could change it to `null` if it's clearer, though